studioShared/lib/token.ts

```diff
@@ -0,0 +1,23 @@
+// Server-Only Token Management
+
+import "server-only";
+import { experimental_taintUniqueValue } from "react";
+
+export const token =
+  process.env.NODE_ENV === "development"
+    ? process.env.SANITY_API_TOKEN_DEV
+    : process.env.SANITY_API_TOKEN_PROD;
```

Comment:
Should this use shared tokens?
```suggestion
    ? process.env.SANITY_SHARED_API_TOKEN_DEV
    : process.env.SANITY_SHARED_API_TOKEN_PROD;
```